scripts/globals/transport.lua

```diff
@@ -33,4 +60,15 @@ tpz.transport.captainMessage = function(npc, triggerID, messages)
     for _, player in pairs(playersInZone) do
         player:showText(player, messages[triggerID])
     end
+end
+
+tpz.transport.dockPorterMhauraMessages = function(npc, triggerID, messages)
+    npc:showText(npc, messages[triggerID])
+    if triggerID == tpz.transport.dockPorterMhauraTrigger.FERRY_ARRIVING_FROM_ALZAHBI or
+        triggerID == tpz.transport.dockPorterMhauraTrigger.FERRY_ARRIVING_FROM_SELBINA then
```

Comment:
Here's a trick you could do to save some lines and space, since your "arriving" and "departing" are all even/odd:

```lua
if (triggerID % 2) == 0 then
    npc:pathThrough(-- arriving)
else
    npc:pathThrough(-- departing)
end
```